client/data/geo/use-geolocation-query.js

```diff
@@ -0,0 +1,8 @@
+import { useQuery } from 'react-query';
+
+export const useGeoLocationQuery = () =>
+	useQuery( 'geo', () =>
+		globalThis
+			.fetch( 'https://public-api.wordpress.com/geo/' )
```

Comment:
It's interesting that we actually need the trailing slash for this endpoint.

Reply:
Yeah I tried to remove it and failed miserably :)